gitops-deployment/README.md

```diff
@@ -9,115 +9,241 @@ For more information about ArgoCD, see the [ArgoCD documentation](https://argo-c
-    roleRef:
-      apiGroup: rbac.authorization.k8s.io
-      kind: ClusterRole
-      name: cluster-admin
-    EOF
-    ```
-
-4. Access the ArgoCD UI and verify Git Repository Connnectivity.
-    
-    The ArgoCD URL can be obtained via the `openshift-gitops` namespace route.
-    ```bash
-    oc get routes -n openshift-gitops
-    ```
-
-    Navigate to that route and login by selecting the OpenShift Login option and providing the clusters credentials.    
-
-## Deploy ArgoCD Applications
-
-### Custom Health Checks
-
-Create the ArgoCD Application for Custom Health Checks:
 ```bash
 cat <<EOF | oc apply -f -
 apiVersion: argoproj.io/v1alpha1
 kind: Application
 metadata:
   name: argocd
-  namespace: openshift-gitops
-  finalizers:
-    - resources-finalizer.argocd.argoproj.io
+  namespace: ${GITOPS_NAMESPACE}
   labels:
     app.kubernetes.io/instance: argocd
   annotations:
     argocd.argoproj.io/sync-options: SkipDryRunOnMissingResource=true
 spec:
   destination:
-    namespace: default
+    namespace: ${GITOPS_NAMESPACE}
     server: 'https://kubernetes.default.svc'
   source:
-    repoURL: 'https://github.com/IBM/ibm-websphere-automation'
+    repoURL: ${SOURCE_REPOSITORY}
     path: gitops-deployment/argocd
-    targetRevision: main
+    targetRevision: ${TARGET_REVISION}
+    helm:
+      valuesObject:
+        gitops:
+          instance: ${GITOPS_INSTANCE}
+          namespace: ${GITOPS_NAMESPACE}
+  syncPolicy:
+    retry:
+      limit: 10
+      backoff:
+        duration: 5s
+        factor: 2
+        maxDuration: 1m
+  project: default
+EOF
+```
+
+#### IBM Licensing Service
+
+Skip this step if the IBM Cloud Pak Foundational Services License Service is already installed on the Red Hat OpenShift cluster that you are installing IBM WebSphere Automation on.
+
+Create the following Argo CD Application to deploy the IBM Licensing Service.
+
+```bash
+export SOURCE_REPOSITORY=https://github.com/IBM/ibm-websphere-automation
+export TARGET_REVISION=<release-version>
+export GITOPS_NAMESPACE=openshift-gitops
+```
+
+```bash
+cat <<EOF | oc apply -f -
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: ibm-licensing
+  namespace: ${GITOPS_NAMESPACE}
+  labels:
+    app.kubernetes.io/instance: ibm-licensing
+  annotations:
+    argocd.argoproj.io/sync-options: SkipDryRunOnMissingResource=true
+spec:
+  destination:
+    namespace: ibm-licensing
+    server: 'https://kubernetes.default.svc'
+  source:
+    repoURL: ${SOURCE_REPOSITORY}
+    path: gitops-deployment/licensing
+    targetRevision: ${TARGET_REVISION}
+  syncPolicy:
+    retry:
+      limit: 10
+      backoff:
+        duration: 5s
+        factor: 2
+        maxDuration: 1m
+  project: default
+EOF
+```
+
+#### IBM Cert Manager
+
+Skip this step if you already have a certificate manager installed on the Red Hat OpenShift cluster that you are installing IBM WebSphere Automation on. If you do not have a certificate manager then you must install one. 
```

Comment:
I think we should say something about being able to use the Red Hat cert manager.